.github/workflows/python-publish.yml

```
@@ -0,0 +1,15 @@
+name: Upload Python Package
+
+on:
+  release:
+    types:
+      - created
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Build and publish to pypi
+      uses: JRubics/poetry-publish@v1
+      with:
+        pypi_token:  ${{ secrets.PYPI_API_KEY }}
```

```suggestion
        pypi_token:  ${{ secrets.PYPI_API_KEY }}

```
Add a newline here plox.